centralledger/chart-handler-admin-transfer/configs/default.json

```diff
@@ -5,7 +5,7 @@
     "PORT": {{ .Values.containers.api.service.ports.api.internalPort }},
     "HOSTNAME": "{{ .Values.ingress.hosts.api }}",
     "MIGRATIONS": {
-        "DISABLED": true,
+        "DISABLED": false,
```

Comment:
This should be `true`.

This is run by an initContainer on the chart-central-service